Tasks/PackerBuildV0/Tests/L0LinuxsBuiltinTemplateAdditionalParameters.ts

```diff
@@ -1,132 +0,0 @@
-import ma = require('vsts-task-lib/mock-answer');
```

Comment:
Hey @bishal-pdMSFT , I noticed that this file isn't being referenced anywhere. There is a typo in the filename ( notice the Linux**s** ). There is another file with the correct name already present. Can you confirm if this isn't required ?

Reply:
If it is not being used anywhere, please delete it

Reply:
done